pkg/commonvpc/common_test_functionality.go

```diff
@@ -47,11 +37,17 @@ const (
 const errString = "err: %s"
 const carriageReturn = "\r"
 
-type VpcGeneralTest struct {
+/*
+tests for the entire flow:
+	- input from config json file
+	- output comparison, for the possible output use-cases
+	- currently comparing only txt output formats
+*/
+
+type VpcTestCommon struct {
```

Comment:
Will it make sense to consider `VpcTestCommon` as just an "abstract" base class, and have an additional `VpcAnalysisTest` struct?

Reply:
Not sure I follow. The delta - fields wise -  between the properties of `VpcTestCommon`  and `VpcAnalysisTest` , at least at the moment, is empty. What are you suggesting exactly? Do define an extra struct of `VpcAnalysisTest`? I actually did so in the beginning, and then removed it since it seemed useless. Or is there something else you are suggesting?

Reply:
Some fields in `VpcTestCommon` are only relevant to analysis (e.g., `Grouping`).
It is therefore weird that `VpcLintTest` for example has to include this field.

Seems to me like a cleaner solution to have a base class with only fields that are relevant to all types of test cases.

Reply:
done